// alchemy-sdk/dist/cjs/index.js
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

var utils = require('./api/utils');
var index = require('./index-be313c8a.js');
require('@ethersproject/bignumber');
require('axios');
require('@ethersproject/abstract-provider');
require('@ethersproject/wallet');
require('@ethersproject/contracts');

function _interopNamespace(e) {
	if (e && e.__esModule) return e;
	var n = Object.create(null);
	if (e) {
		Object.keys(e).forEach(function (k) {
			if (k !== 'default') {
				var d = Object.getOwnPropertyDescriptor(e, k);
				Object.defineProperty(n, k, d.get ? d : {
					enumerable: true,
					get: function () { return e[k]; }
				});
			}
		});
	}
	n["default"] = e;
	return Object.freeze(n);
}

var utils__namespace = /*#__PURE__*/_interopNamespace(utils);



exports.Utils = utils__namespace;
exports.Alchemy = index.Alchemy;
Object.defineProperty(exports, 'AssetTransfersCategory', {
	enumerable: true,
	get: function () { return index.AssetTransfersCategory; }
});
Object.defineProperty(exports, 'AssetTransfersOrder', {
	enumerable: true,
	get: function () { return index.AssetTransfersOrder; }
});
exports.Contract = index.Contract;
exports.ContractFactory = index.ContractFactory;
Object.defineProperty(exports, 'Network', {
	enumerable: true,
	get: function () { return index.Network; }
});
Object.defineProperty(exports, 'NftExcludeFilters', {
	enumerable: true,
	get: function () { return index.NftExcludeFilters; }
});
Object.defineProperty(exports, 'NftTokenType', {
	enumerable: true,
	get: function () { return index.NftTokenType; }
});
Object.defineProperty(exports, 'RefreshState', {
	enumerable: true,
	get: function () { return index.RefreshState; }
});
Object.defineProperty(exports, 'TokenBalanceType', {
	enumerable: true,
	get: function () { return index.TokenBalanceType; }
});
exports.Wallet = index.Wallet;
exports.fromHex = index.fromHex;
exports.isHex = index.isHex;
exports.setLogLevel = index.setLogLevel;
exports.toHex = index.toHex;
